docs/dev/src/SUMMARY.md

```diff
@@ -0,0 +1,41 @@
+# Summary
+
+[Introduction](./intro.md)
+[Contribution Procedure](./contribution.md)
+
+---
+
+# Building and debugging RisingWave
+
+- [How to build and run RisingWave](./build-and-run/intro.md)
```

Comment:
The title looks too long to me. What about "Getting Started"/"Building and Running"?